executor.go

```diff
@@ -171,20 +171,23 @@ func (e *Executor) verifySubjectAgainstReferrers(ctx context.Context, task *exec
 	err := e.Store.ListReferrers(ctx, artifact, referenceTypes, func(referrers []ocispec.Descriptor) error {
 		for _, referrer := range referrers {
 			results, err := e.verifyArtifact(ctx, repo, task.artifactDesc, referrer, evaluator)
-			if err != nil && !errors.Is(err, errSubjectPruned) {
-				return err
-			}
-
 			artifactReport := &ValidationReport{
 				Subject:  artifact,
 				Results:  results,
 				Artifact: referrer,
 			}
+			if err != nil {
```

Comment:
code style: please follow the standard code pattern of
```go
xxx, err := xxx()
if err != nil {
  // error handling
}

// proceed
```

Reply:
Updated.